locales/base/translation.json

```diff
@@ -2185,5 +2185,33 @@
     "discover": {
       "tabName": "Discover"
     }
+  },
+  "points": {
+    "title": "Valora Points",
+    "activity": "Activity",
+    "infoCard": {
+      "title": "More ways to earn points coming soon 🚀",
```

Comment:
i haven't actually added an emoji icon like this to the app before... doing it this way felt kinda wrong, but also maybe correct? let me know if it should be an icon that we check in instead!

Reply:
I guess both iOS and Android share most emojis, so we're mostly fine.
Otherwise it shows squares :D